WalletWasabi.Fluent/Views/Wallets/WalletSettingsView.axaml

```diff
@@ -24,6 +24,12 @@
           <ToggleSwitch IsChecked="{Binding AutoCoinJoin}" />
         </DockPanel>
 
+        <StackPanel Spacing="10"
+            ToolTip.Tip="Won't coinjoin automatically if there are less than this much non-private coins in the wallet.">
+          <TextBlock Text= "Coinjoin Threshold (BTC)" />
+          <TextBox Text="{Binding PlebStopThreshold}" />
+        </StackPanel>
+
```

Comment:
Consider using `DockPanel`

```suggestion
        <DockPanel IsVisible="{Binding !IsWatchOnly}"
            ToolTip.Tip="Won't coinjoin automatically if there are less than this much non-private coins in the wallet.">
          <TextBlock Text= "Coinjoin Threshold (BTC)" />
          <TextBox Text="{Binding PlebStopThreshold}" />
        </DockPanel >

```

![image](https://user-images.githubusercontent.com/45069029/157440012-0824a5b5-f69e-462b-8977-7f2362b0f5b2.png)

Reply:
Done. When the "coma" error message is shown, the textbox will be a little wider, but it's still good IMO

Reply:
> When the "coma" error message is shown, the textbox will be a little wider, but it's still good IMO

No, that is bad and annoying, please fix it.

Reply:
Fixed